lib/notification-support.ts

```diff
@@ -27,11 +27,20 @@ function supportsDarwinNotifications() {
 }
 
 function supportsWindowsNotifications() {
+  // The Windows Notification APIs we use are only available on Windows 10+, and
+  // some of them are limited before the Creators Update (build 15063).
+  // See: https://docs.microsoft.com/en-us/uwp/api/windows.ui.notifications.toastnotification.tag?view=winrt-22621#remarks
+  const CreatorsUpdateBuildNumber = 15063
+
   const versionComponents = os.release().split('.')
   const majorVersion = parseInt(versionComponents[0], 10)
+  const buildNumber =
+    versionComponents.length >= 3
+      ? parseInt(versionComponents[2], 10)
+      : CreatorsUpdateBuildNumber
```

Comment:
This looks like you are defaulting to the creators update build number when you cannot find the build number. Wouldn't you want to default to not supporting if we couldn't determine it?

Or.. maybe there is some context I am missing in which case maybe some comment about what version have build numbers and which do not. :)

Reply:
The idea is that right now all known build numbers look like `X.Y.Z`, so in case that changes in the future (where these notifications should still be supported), let's assume they are supported.

Does that make sense? 😳

Reply:
> K. I think I got it... We are saying that we know there are some windows 10 that don't work, because they don't have the creators update build. Thus, if we can determine it doesn't work than disable it... but if we cannot determine it doesn't work err on the side of allowing it. My one last question then is, there are no negative consequences in windows for allowing it when it doesn't work?

That's unknown, because _in theory_ the only versions of Windows that could fall under that scenario are future versions with different version naming/scheme.

All the Windows versions supported by Desktop as of today (Window 7 to 11) use that `X.Y.Z` version format, so I assume future versions will still support the existing APIs (and in any case will add new ones, but won't stop supporting the old ones).